crates/evm/src/lib.rs

```diff
@@ -137,26 +135,15 @@ pub trait ConfigureEvmEnv: Send + Sync + Unpin + Clone + 'static {
     );
 
     /// Fill [`BlockEnv`] field according to the chain spec and given header
-    fn fill_block_env(
-        &self,
-        block_env: &mut BlockEnv,
-        chain_spec: &ChainSpec,
-        header: &Header,
-        after_merge: bool,
-    ) {
-        let coinbase = block_coinbase(chain_spec, header, after_merge);
-        Self::fill_block_env_with_coinbase(block_env, header, after_merge, coinbase);
+    fn fill_block_env(&self, block_env: &mut BlockEnv, header: &Header, after_merge: bool) {
+        Self::fill_block_env_inner(block_env, header, after_merge);
     }
 
     /// Fill block environment with coinbase.
-    fn fill_block_env_with_coinbase(
-        block_env: &mut BlockEnv,
-        header: &Header,
-        after_merge: bool,
-        coinbase: Address,
-    ) {
+    /// TODO: fix
+    fn fill_block_env_inner(block_env: &mut BlockEnv, header: &Header, after_merge: bool) {
```

Comment:
@mattsse better name for this?

Reply:
remove it and reuse the existing `fn fill_block_env`

Reply:
https://github.com/paradigmxyz/reth/pull/9322